shared/src/native-src/string.cpp

```diff
@@ -152,6 +153,13 @@ namespace shared {
         return str.size() >= prefix.size() && str.compare(0, prefix.size(), prefix) == 0;
     }
 
+    WSTRING to_lower(const WSTRING& str)
+    {
+        WSTRING lower(str);
+        transform(lower.begin(), lower.end(), lower.begin(), ::towlower);
+        return lower;
+    }
```

Comment:
While this is correct for a certain subset of `wchar`, this implementation has several issues. More details here: https://devblogs.microsoft.com/oldnewthing/20241007-00/?p=110345

Reply:
Thanks @dmehala I've opted to leave it out for the time being as I'll need a bit more time understanding how to address this best (or find an example that does it 😛 )